Installer/51-gcadapter.rules

```diff
@@ -0,0 +1 @@
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="057e", ATTRS{idProduct}=="0337", MODE="0666"
```

Comment:
Unless there's a group for Dolphin users, yes. Some distros have special groups like `device` or `dialout` (which is a bit antiquated, but still used), but nothing universal afaik.